src/amplitude-client.js

```diff
@@ -702,15 +732,15 @@ AmplitudeClient.prototype._saveReferrer = function _saveReferrer(referrer) {
 AmplitudeClient.prototype.saveEvents = function saveEvents() {
   try {
     if (AsyncStorage) {
-      AsyncStorage.setItem(this.options.unsentKey, JSON.stringify(this._unsentEvents));
+      AsyncStorage.setItem(this.options.unsentKey + this._storageSuffix, JSON.stringify(this._unsentEvents));
```

Comment:
not blocking but in terms of API I feel like `_setInStorage` should encapsulate the `AsyncStorage` logic inside it, so that whoever calls _setInStorage doesn't have to worry about checking Async etc etc you know what I mean?